### ISSUE TITLE
Add moonshine.dev on main panel screen

### ISSUE THREAD

**Issue by piotrzarzycki21** [Moonshine-IDE/Super.Human.Portal#62]
Please add a tile for Moonshine.dev on the Getting Started page here:

It should read something like:

Thinking about a new web, desktop, or mobile app for your business?  Create a User Interface mockup and initial working design both at once with Moonshine.dev Use the drag-and-drop canvas visual editor in your browser.   Or ask the AI to create the form to get you started quickly

**Comment by JoelProminic** [comment]
@piotrzarzycki21, I thought this was based on the configured categories.  I would expect to need to add the category and at least one document in domino-49 to support this.

**Comment by piotrzarzycki21** [comment]
@JoelProminic To me it would be the best if we could do that - provide me category and I will test how it looks like. I just thought it is something which we would like to have fast and exposed a bit separately than others.

**Comment by piotrzarzycki21** [comment]
@JoelProminic I can propose following design. Category item should gain new field called "link" - where there will be direct link to some resources - in our case www.moonshine.dev.

Icon should contains either google icon or link to specific png in our case it will be: https://github.com/Moonshine-IDE/MoonshineWebsite/blob/master/images/moonshine-logo-circle.png

We will have something like this + "link" field:
<img width="1058" alt="Screenshot 2024-09-10 at 09 43 23" src="https://github.com/user-attachments/assets/91b9a11b-427c-4139-9347-5d4fd35bd265">

Do not deploy current version of portal until we agreed on that and I make the changes. If this design is ok, please provide me new database.

**Comment by piotrzarzycki21** [comment]
@JoelProminic It looks like if I get link as Icon I have some problems with loading that image - it's being blocked. If we wanted to have moonshine logo we need to either store it or just use google icon. What do you suggest ?

**Comment by piotrzarzycki21** [comment]
@JoelProminic code for Moonshine.dev tile has been pushed.

**Comment by JoelProminic** [comment]
I got started on this today, but I got pulled into another task while I was testing.

Here is the Moonshine.dev writeups from @piotrzarzycki21's earlier screenshot.  I used this in the central copy of Super.Human.Portal:

>Thinking about a new web, desktop, or mobile app for your business?  Create a User Interface mockup and initial working design both at once with Moonshine.dev.  Use the drag-and-drop canvas visual editor in your browser or ask the AI to create the form to get you started quickly.

The new category document looks like this:
![image](https://github.com/user-attachments/assets/e57058a1-4bed-4f5c-a3ab-9e06eb8aaf9c)

Let me know if this does not match what you expected.

**Comment by piotrzarzycki21** [comment]
@JoelProminic This is exactly what I wanted! Let's proceed farther.

**Comment by piotrzarzycki21** [comment]
@JoelProminic I was having typo parsed category. Not sure how it was working at my sight before. Fresh build have my fix now.

**Comment by JoelProminic** [comment]
Ah, `Link` vs `link`?   I used the capital for consistency with the other elements.  Sorry, I meant to remind you about this.

Unfortunately, I'm seeing the same behavior with [this build](https://github.com/Moonshine-IDE/Super.Human.Portal/actions/runs/10937426398).
- The icon is missing, and the code is trying to treat the image link as an icon reference.
- The category does not work as a link

![image](https://github.com/user-attachments/assets/b986b0bc-236f-466b-9b04-801066ef4999)

I don't see any obvious errors on the browser console.  I did a hard refresh on the application and double checked by build installation.

**Comment by piotrzarzycki21** [comment]
What about Network tab ? Do you see problem with loading Moonshine image ? 

I will deploy tomorrow build from GH and see if it fails.

**Comment by JoelProminic** [comment]
I created a release candidate [here](https://github.com/Moonshine-IDE/Super.Human.Portal/releases/tag/untagged-12bf3cc9bfd7129f4d61) to test the updates.   

I confirmed that the icon and link logic are fixed:
![image](https://github.com/user-attachments/assets/b24eaf4b-89da-4305-9568-761e2294ffbd)

I also confirmed that I can set a link for another category in SuperHumanPortal.nsf.   I can also set a custom icon image, as long as it is already deployed to the server.

I'll work towards releasing v1.1.0.
